src/fixture/generators/string/index.ts

```diff
@@ -1,6 +1,7 @@
 import { ZodString } from '@/internal/zod';
 import { Generator } from '@/transformer/generator';
 import type { Runner } from '@/transformer/runner';
+import RandExp from 'randexp';
```

Comment:
You might consider using:
```ts
import { randexp } from 'randexp'
```

It has a nicer syntax:
```ts
randexp(pattern)

new RandExp(pattern).gen()
```

Reply:
I kept the same syntax because we can also configure it with our seed logic.